python/src/energuide/extractor.py

```diff
@@ -79,6 +81,10 @@ def __iter__(self):
 
 
 def write_data(data: typing.Iterable[reader.InputData], output_path: str) -> None:
-    output_data = _SerializableGenerator(data)
-    with open(output_path, 'w') as output_file:
-        json.dump(output_data, output_file)
+    with zipfile.ZipFile(output_path, mode='w') as output_zip:
+        for blob in data:
+            blob_id = blob.get('BUILDER')
+            if blob_id:
+                output_zip.writestr(blob_id, json.dumps(blob))
+            # else:
+                # log(f'Invalid data: {blob}, must contain field \'BUILDER\'')
```

Comment:
🔥  ?

Reply:
Mostly just there to remind me/us that once logging is in place, this is somewhere that will immediately take advantage of it.

If you prefer not to include things like that, I have no problems with removing it.